README.md

```diff
@@ -0,0 +1,136 @@
+HighLine
+========
+
+by James Edward Gray II
```

Comment:
Great!

You can see how it will look like...
at GitHub - https://github.com/abinoam/highline/tree/develop-3
at Yard (Rubydoc.info) - http://www.rubydoc.info/github/abinoam/highline/master (my fork link as it's not merged yet)

I'll do some modifications more like... The tag {HighLine} is good only for yard. As README.md is the door at GitHub, I'll change this to a link to the Rubydoc.info documentation instead. So it renders properly on GitHub.